www/viewgame

```diff
@@ -2846,12 +2846,9 @@ dispTags();
                         if (!$cssOverride) {
                         ?>
                         <script nonce="<?php global $nonce; echo $nonce; ?>">
-                            xmlSend("ifwiki-check?ifid=<?php echo $jifid ?>", null,
-                                function(doc) {
-                                    if (xmlChildText(doc, "exists") == "yes") {
-                                        document.getElementById("ifwiki-link").style.display = "initial";
-                                    }
-                                }, null);
+                            fetch("ifwiki-check?ifid=<?php echo $jifid ?>").then(() => {
+                                document.getElementById("ifwiki-link").style.display = "initial";
+                            });
```

Comment:
This isn't checking the status. <http://localhost:8080/viewgame?id=qvcdjqvl43b0whlx> is showing an IFWiki link, but it shouldn't.